tests/D2L.CodeStyle.Analyzers.Test/Specs/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -118,7 +122,19 @@ public sealed class SomeClass {
 			new SomeClass( p, p );
 			/* TooManyUnnamedArgs */ new SomeClass( p, p, p, p, p ) /**/;
 			new SomeClass( a1: 1, p, p, p, p );
-			#endregion
-		}
-	}
+            #endregion
+
+            #region expressions should not trigger named argument diagnostics
+            // See: https://stackoverflow.com/a/10133102
+            System.Linq.Expressions.Expression<Func<int>> expression5args = () => _arg5_ret(p, p, p, p, p);

```

Comment:
You could probably reuse the existing helpers with `Expression<Action>` instead of `Expression<int>` :)

Reply:
Cool, I was wondering how to make a void function pointer. I used the int returning methods for testing nested calls though, so as of now they serve a purpose